x-pack/plugins/security_solution/public/management/pages/event_filters/index.tsx

```diff
@@ -20,3 +20,5 @@ export const EventFiltersContainer = () => {
   );
 };
 export { EventFiltersService } from './types';
+export { EventFiltersListPageState } from './types';
```

Comment:
Out of curiosity (don't need to be changed), Is there a reason why you do this instead of:
```
export { 
  EventFiltersService,
  EventFiltersListPageState,
  EventFiltersListPageData,
} from './types';
```

Reply:
I think this might have been my editor when I did the refactor - they should be collapsed into a single `export` as you noted. I'll fix in my next PR.

_(Also - In reality, this file `index.tsx` does not have the need to export any of the types)_